.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+  - repo: https://github.com/pycqa/isort
```

Comment:
Are we asking to developers to run these two commands to setup their local env?:

```
pip install tensorflow
pip install -e ".[tests]" --progress-bar off --upgrade
```

In that case we could set `repo: local` and remove the extra bazel cache

Reply:
One of the advantage to handling the lint tool versioning in the `pre-commit` config  Is that we could use the autoupdate PR if we want to use https://pre-commit.ci/.

There is also an alternative Github Action for managing the update:
https://github.com/marketplace/actions/pre-commit-autoupdate

For more context, also if it is a little bit outdated now, see:
https://browniebroke.com/blog/gh-action-pre-commit-autoupdate/

Reply:
Please check again this comment.